README.md

```diff
@@ -1,100 +0,0 @@
-## DGL Implementation of the MixHopGCN Paper
-
-This DGL example implements the GNN model proposed in the paper [MixHop: Higher-Order Graph Convolution Architectures via Sparsified Neighborhood Mixing](https://arxiv.org/abs/1905.00067). The author's codes of implementation is in [here](https://github.com/samihaija/mixhop).
-
-## Example implementor
-
-This example was implemented by [xnuohz](https://github.com/xnuohz) during his intern at the AWS Shanghai AI Lab.
-
-### Requirements
-The codebase is implemented in Python 3.6.11. package versions used for devalopment are just below.
-
-```
-dgl 0.5.2
-numpy 1.19.4
-pandas 1.1.4
-tqdm 4.53.0
-torch 1.7.0
-```
-
-### The graph dataset used in this example
-
-The DGL's built-in Cora, Pubmed and Citeseer GraphDataset. Dataset summary:
-
-| Datset | Nodes | Edges | Feats | Classes | Train | Val | Test |
-| :-: | :-: | :-: | :-: | :-: | :-: | :-: | :-: |
-| Citeseer | 3,327 | 9,228 | 3,703 | 6 | 120 | 500 | 1000 |
-| Cora | 2,708 | 10,556 | 1,433 | 7 | 140 | 500 | 1000 |
-| Pubmed | 19,717 | 88,651 | 500 | 3 | 60 | 500 | 1000 |
-
-### Usage
-
-###### Dataset options
-```
---dataset     string     The graph dataset name.                   Default is 'Cora'.
```

Comment:
I guess we can directly use the standard types in python, i.e. `str`, `int`, `float`.

Reply:
Done.